### ISSUE TITLE
Test automation parameter change

### ISSUE THREAD

**Issue by kdavis-mozilla** [mozilla/DeepSpeech#201]
In light of the last run it'd be interesting to try some new parameter values.

- To speech up convergence, try a larger `ds_learning_rate` 0.01
- To speed things up try a bigger `ds_display_step` 10
- To test if we can overfit `ds_dropout_rate` 0.0 (This will likely increase validation & test WER)
- Maybe decrease `ds_default_stddev` to something like `0.05` too?

From the DeepSpeech paper[[1](https://arxiv.org/abs/1412.5567)] the state

> ...we train our RNNs on all the datasets (more than 7000 hours) listed in Table 2. Since we train for 15 to 20 epochs...

So their network sees `15*7000=105000` to `20*7000=140000` hours of data during training. As TED is about `300` hours this implies that we should run about `350` to `467` epochs! That's alot! I'd suggest we start with something more reasonable.

- To get a better convergence try a larger `ds_training_iters` 30

**Comment by lissyx** [comment]
Given the current run time I would think we should try the options that increase velocity first, this will allow us to iterate faster :-)

**Comment by tilmankamp** [comment]
We should also consider a larger ```validation_step```.

**Comment by kdavis-mozilla** [comment]
`ds_validation_step` is currently 15, I thought keeping one validation in the middle and one at the ends seemed ok.

**Comment by kdavis-mozilla** [comment]
@lissyx Which options do you think would be best to increase velocity?

**Comment by lissyx** [comment]
that's the one billion dollar question :). I guess we first need to understand what is going on really regarding GPU usage, and what you saw of maybe CPU limitation is interesting. But I guess that moving to reduce the non critical computations can probably help, especially display_step: it is easy fix, easy win, huge reward

**Comment by kdavis-mozilla** [comment]
I like the `display_step` idea. It's simple. It will work, and it's trivial to implement. :-)

**Comment by tilmankamp** [comment]
@kdavis-mozilla Regarding ```ds_validation_step```: You are right. Just wasn't sure, if we override it by an environment variable setting in our script.

**Comment by lissyx** [comment]
We should also move the Epoch print out of the validation_step condition, so we keep getting some progress feedback :)

**Comment by tilmankamp** [comment]
@lissyx You mean out of the display_step condition? So that every epoch gets printed, right?

**Comment by lissyx** [comment]
Exactly!

**Comment by tilmankamp** [comment]
@lissyx @kdavis-mozilla Started #205 for this.
